Automattic/vip-go-mu-plugins
# Search: Enable admin UI

## Description
This PR is dependent on https://github.com/Automattic/ElasticPress/pull/153 and will re-enable the UI with customizations.

## Changelog Description

### Plugin Updated: Enterprise Search

Expose Search UI to wp-admin.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Go to wp-admin and see "Enterprise Search" submenu revealed
2) Observe admin footer text has changed as well
